src/KlarnaOrderManagement.php

```diff
@@ -162,6 +163,34 @@ public function enqueue_admin() {
 		if ( isset( $this->metabox ) && method_exists( $this->metabox, 'maybe_localize_script' ) ) {
 			$this->metabox->maybe_localize_script( 'kom-admin-js' );
 		}
+
+		wp_enqueue_style( 'kom-admin-style', KLARNA_ORDER_MANAGEMENT_CHECKOUT_URL . '/assets/css/klarna-order-management.css', array(), KLARNA_ORDER_MANAGEMENT_VERSION );
+
+		// Script Params.
+		$params = array(
+			'ajax_url'                                => admin_url( 'admin-ajax.php' ),
+			'with_return_fee_text'                    => __( 'minus a return fee of', 'klarna-order-management' ),
+			'refund_amount_less_than_return_fee_text' => __( 'Refund amount is less than the return fee.', 'klarna-order-management' ),
+		);
+
+		// Checkout script.
+		wp_register_script(
+			'kom-admin-js',
+			KLARNA_ORDER_MANAGEMENT_CHECKOUT_URL . '/assets/js/klarna-order-management.js',
+			array( 'jquery' ),
+			KLARNA_ORDER_MANAGEMENT_VERSION,
+			true
+		);
+
+		// Localize the script and add the params.
+		wp_localize_script(
+			'kom-admin-js',
+			'kom_admin_params',
+			$params
+		);
+
+		// Enqueue the script.
+		wp_enqueue_script( 'kom-admin-js' );
```

Comment:
Issue: From what i can tell, the script and CSS file is loaded on every admin page with this logic. There should be a check to only do this on a edit order page, since that is currently the only place this is done.

Also from what i see we now register the script on line 177, but the metabox is trying to localize the script on row 164, and on row 186 you are localizing the script again. Does this not break the logic for the metabox toggle right now? As it is right now, the metabox script is enqueued by the OrderMetabox parent class, since we set the script as a dependency in the [constructor](https://github.com/krokedil/klarna-order-management/blob/67989bd201fe10552697e9307ced8757b9f6a1a6/src/MetaBox.php#L45).